tests/WebAssertTest.php

```diff
@@ -982,6 +982,81 @@ public function testElementAttributeExists()
         );
     }
 
+    public function testElementAttributeNotExistsValid()
```

Comment:
This test name doesn't resemble naming convention of test for `elementAttributeNotExists` method.